code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -322,12 +324,20 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/process()
 	var/turf/location = get_turf(src)
-	var/mob/living/M = loc
+	var/mob/living/M
+	if(inside_item)
+		if(loc)
```

Comment:
Making sure that it actually has a loc. Just in case.

If you mean the whole block, I needed to make sure that, if the cigarette was in the contents of another item, the mob would be whatever is holding that item.